packages/ckeditor5-dev-tests/bin/test-travis.sh

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env bash
 
+set -e
```

Comment:
Please add info why `set -e` - I had to google it so others might as well as we do not code in Linux shell that much :)

Reply:
Done.